front-end/src/Components/HeartHealth.js

```diff
@@ -1,10 +1,39 @@
-import heartSolid from "../assets/heart-solid.png";
-import heartOutline from "../assets/heart-regular.png";
+
+import Modal from "react-bootstrap/Modal";
+import { useState } from "react";
+import "bootstrap/dist/css/bootstrap.min.css";
+
 
 function HeartHealth({ snackHealth }) {
+
+  // console.log("LINE 6:",snackHealth.fiber)
+
+  const [isOpen, setIsOpen] = useState(true);
+
+  // const showModal = () => {
+  //   setIsOpen(true);
+  // };
+
+  const hideModal = () => {
+    setIsOpen(false);
+  };
+
+
   return (
+
     <>
-      <p>????</p>
+    <Modal show={isOpen} onHide={hideModal}>
+        <Modal.Header>
+          <Modal.Title>Health Status</Modal.Title>
+        </Modal.Header>
+        <Modal.Body>
+        {snackHealth.fiber > 5 || snackHealth.protein > 5 && snackHealth.added_sugar < 5   ? 'This snack is UNHEALTHY' : 'This snack is HEALTHY'}
```

Comment:
We should start trying to space out our JS in JSX {this} is harder to read than { this } - also mind the indentation on 30